lib/gems/pending/appliance_console/key_configuration.rb

```diff
@@ -63,6 +66,17 @@ def activate
       end
     end
 
+    def save_temp_key
+      FileUtils.mv(KEY_FILE_TEMP, KEY_FILE)
```

Comment:
Should we use `:force => true` here? (ref: https://ruby-doc.org/stdlib-2.3.1/libdoc/fileutils/rdoc/FileUtils.html#method-c-mv)